packages/app-project/server/server.js

```diff
@@ -31,6 +34,18 @@
   server.use(Sentry.Handlers.requestHandler())
   setLogging(server)
 
+  server.get('/images/:fileName', (req, res) => {
+    const [fileName, fileType] = req.params.fileName.split('.');
+    const filePath = path.join(process.cwd(), `/public/images/${fileName}.${fileType}`);
+    try {
+      const imageBuffer = fs.readFileSync(filePath);
```

Comment:
This actually opens us up to directory traversal attacks, via the URL in the browser location bar. It would be interesting to see if this allows an end user to display `next.config.js` in their browser, by editing an image URL to point to the config file. 

Since NextJS already has built-in support for static files in the `public` directory, you don’t need to write your own custom handler for images.

Reply:
@eatyourgreens - good catch on this. Was able to reuse the /projects/assets URL.